### DISCUSSION TITLE
Keyboard Shortcut for Session Management

### DISCUSSION THREAD

**Discussion by idesmar** [gioxx/MarvellousSuspender#111] [Ideas]
Firstly, I would like to thank gioxx for reviving and slowly regaining the trust for this extension. Your efforts are very well appreciated.

Is it possible to create a keyboard shortcut for Session Management? It would be a great user experience if one can restore the last session just by using the keyboard.

Cheers!

**Comment by gioxx** [comment]
Ciao @idesmar, good question and good suggestion.
We will evaluate this enhancement for a next version of TMS, I have opened a new issue to track this request: https://github.com/gioxx/MarvellousSuspender/issues/116

Thanks!
Giovanni.
